### DISCUSSION TITLE
Status shows none, but it's working

### DISCUSSION THREAD

**Discussion by amriamiruddin** [DNSCrypt/dnscrypt-proxy#2603] [Q&A] [answered]
```sh
$ ./dnscrypt-proxy -version
2.1.5

$ ./dnscrypt-proxy -check
[2024-03-22 19:31:06] [NOTICE] dnscrypt-proxy 2.1.5
[2024-03-22 19:31:06] [NOTICE] Source [public-resolvers] loaded
[2024-03-22 19:31:06] [NOTICE] Source [relays] loaded
[2024-03-22 19:31:06] [NOTICE] Configuration successfully checked

$ ./dnscrypt-proxy -resolve example.com
Resolving [example.com] using 127.0.0.1 port 53

Resolver      : 74.125.190.141

Canonical name: example.com.

IPv4 addresses: 93.184.216.34
IPv6 addresses: 2606:2800:220:1:248:1893:25c8:1946

Name servers  : a.iana-servers.net., b.iana-servers.net.
DNSSEC signed : yes
Mail servers  : 1 mail servers found

HTTPS alias   : -
HTTPS info    : -

Host info     : -
TXT records   : v=spf1 -all, wgyf8z8cgvm2qmxpnbnldrcltvk4xqfn
```

In brew services list, Status shows none, but it's working. Do i need root also to run it ? without run it's not working

Sometimes it's even says error the status but still working

Occurs on MacOS Monterey 12.7.4

```sh
$ sudo brew services start dnscrypt-proxy
$ brew services list

Name              Status User File
dnscrypt-proxy    none   root
```

Expected:

```sh
Name              Status User File
dnscrypt-proxy    start   root
```

**Comment by jedisct1** [comment, marked as answer]
`./dnscrypt-proxy` suggests that you manually installed it, and are running it from the directory it was extracted to.

`brew services` is a command that refers to packages installed using Homebrew.

Do you have multiple conflicting installations?

To install the service from a local directory, the documentation says that the command are

```sh
./dnscrypt-proxy -service install
./dnscrypt-proxy -service start
```

**Reply by amriamiruddin** [reply]
Do you have multiple conflicting installations? 

I installed it using homebrew only, not sure if they are conflicting, because i reinstall it by removing the first installed one then i brew install after.

Okay i'll try to installed it manually as you suggested.

**Reply by amriamiruddin** [reply]
It's working like a charm now, I'm installing manually. Thank you
